### PR TITLE
Try .NET updates -- system.collections.generic.list-1.sort

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
samples PR is https://github.com/dotnet/samples/pull/1740

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
@mairaw, @BillWagner -- Maira, Bill, please review these changes.

**Comment by WilliamAntonRohm** [comment]
@mairaw, @BillWagner -- Maira, Bill, please review these changes.

File: xml/System.Collections.Generic/List`1.xml [hunk 1]
```diff
@@ -3618,9 +3618,12 @@ Public Function StartsWith(e As Employee) As Boolean
...
+ 
  The following example adds some names to a `List<String>` object, displays the list in unsorted order, calls the <xref:System.Collections.Generic.List%601.Sort%2A> method, and then displays the sorted list.  
   
- [!code-csharp[System.Collections.Generic.List.Sort#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.collections.generic.list.sort/cs/Sort1.cs#2)]
+ [!code-csharp-interactive[System.Collections.Generic.List.Sort#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.collections.generic.list.sort/cs/Sort1.cs#2)]
  [!code-vb[System.Collections.Generic.List.Sort#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.collections.generic.list.sort/vb/Sort1.vb#2)]  
   
  The following code demonstrates the <xref:System.Collections.Generic.List%601.Sort> and <xref:System.Collections.Generic.List%601.Sort%28System.Comparison%7B%600%7D%29> method overloads on a simple business object. Calling the <xref:System.Collections.Generic.List%601.Sort> method results in the use of the default comparer for the Part type, and the <xref:System.Collections.Generic.List%601.Sort%28System.Comparison%7B%600%7D%29> method is implemented by using an anonymous method.
```
Comment (mairaw) [inline xml/System.Collections.Generic/List`1.xml]: the sample on line 3638 could also be interactive

Comment (WilliamAntonRohm) [reply xml/System.Collections.Generic/List`1.xml]: Added here and in samples PR

**Comment by WilliamAntonRohm** [comment]
@mairaw -- please review.

**Pull request merged**